src/content/history/index.md

```diff
@@ -16,10 +16,11 @@ Forks are when major technical upgrades or changes need to be made to the networ
 When upgrades are needed in traditional, centrally-controlled software, the company will just publish a new version for the end-user. Blockchains work differently because there is no central ownership. [Ethereum clients](/developers/docs/nodes-and-clients/) must update their software to implement the new fork rules. Plus block creators (miners in a proof-of-work world, validators in a proof-of-stake world) and nodes must create blocks and validate against the new rules. [More on consensus mechanisms](/developers/docs/consensus-mechanisms/)
 
 These rule changes may create a temporary split in the network. New blocks could be produced according to the new rules or the old ones. Forks are usually agreed upon ahead of time so that clients adopt the changes in unison and the fork with the upgrades becomes the main chain. However, in rare cases, disagreements over forks can cause the network to permanently split – most notably the creation of Ethereum Classic with the [DAO fork](#dao-fork).
-
 </ExpandableCard>
 
-Looking for future protocol upgrades? [Learn about upcoming upgrades to Ethereum](/upgrades/).
+Skip straight to information about some of the particularly important past upgrades: [The Beacon Chain](/history/beacon-chain/index.md); [The Merge](/history/merge); and [EIP-1559](#london)
```

Comment:
Can I suggest we separate adding new roadmap content from re-structuring? 

Too many considerations that we'd need to think through before executing a re-structuring like this. I've raised these concerns on a previous roadmap call and don't believe they've been adequately addressed to move forward with the restructuring yet. So, moving the current upgrade pages to history is blocking this PR IMO. 

- Do we move all translated versions of these pages too?
- Have we updated Crowdin accordingly?
- Are there areas we should pay for translation (like the history page) to ensure the structure is kept in-tact in non-English languages?
- Have we added appropriate re-directs?
- And, thinking bigger picture, I'm not sold we should we be moving these pages from a technical perspectives due to the strain of creating re-directs across 52 languages every time we call an upgrade 'done'. 
- Finally, the overhead of the above for every time we decide to move one of these pages, seems not worth it to me. 
 
I think this is still an open discussion, but this PR should separated into two tasks IMO.